src/representation_theory/group_cohomology_resolution.lean

```diff
@@ -216,10 +218,67 @@ def of_mul_action_basis  :
   basis (fin n → G) (monoid_algebra k G) (of_mul_action k G (fin (n + 1) → G)).as_module :=
 @basis.map _ (monoid_algebra k G) (monoid_algebra k G ⊗[k] ((fin n → G) →₀ k))
   _ _ _ _ _ _ (@algebra.tensor_product.basis k _ (monoid_algebra k G) _ _ ((fin n → G) →₀ k) _ _
-  (fin n → G) (⟨linear_equiv.refl k _⟩)) (of_mul_action_basis_aux k G n)
+  (fin n → G) ⟨linear_equiv.refl k _⟩) (of_mul_action_basis_aux k G n)
 
 lemma of_mul_action_free :
   module.free (monoid_algebra k G) (of_mul_action k G (fin (n + 1) → G)).as_module :=
 module.free.of_basis (of_mul_action_basis k G n)
 
+end basis
+section differential
+variables (k G)
+
+/- This should maybe be expressed as a nerve... -/
+/-- The simplicial `G`-action sending `[n]` to `Gⁿ⁺¹` equipped with the diagonal action of `G`. -/
+def _root_.simplicial_object.standard_resolution [group G] :
```

Comment:
As @joelriou suggested, this name is maybe not optimal. It should have `group` in the name somewhere.